web-bundle/src/main/java/com/graphhopper/replica/ReplicaCustomSpeedsFootTagParser.java

```diff
@@ -54,7 +59,23 @@ public void handleWayTags(int edgeId, EdgeIntAccess edgeIntAccess, ReaderWay way
     // Overrides speed with custom speed, if one exists
     void setSpeed(int edgeId, EdgeIntAccess edgeIntAccess, double speed, ReaderWay way) {
         if (getCustomSpeedIdType(idToCustomSpeeds) == CustomSpeedId.IdType.SEID) {
-            throw new NotImplementedException("SEID-based custom speeds are not yet implemented for foot!");
+            ImmutableMap<String, Double> stableEdgeIdToMaxSpeed = getStableEdgeIdBasedCustomSpeeds(idToCustomSpeeds);
+
+            String forwardStableEdgeIdForEdge = stableEdgeIdManager.getStableEdgeIdForEdge(false, edgeId);
+            Double forwardSpeed = stableEdgeIdToMaxSpeed.get(forwardStableEdgeIdForEdge);
+            if (forwardSpeed != null) {
+                setSpeed(edgeId, edgeIntAccess, true, false, forwardSpeed);
+            } else {
+                setSpeed(edgeId, edgeIntAccess, true, false, speed);
```

Comment:
Unlike the bike + truck speed parsers, it's not assumed that a (non-custom) speed has already been applied, so we explicitly set the speed of the edge whether or not custom speeds exist for it

Reply:
thanks for context on this. I'd say add a comment as well so it's extra clear in code

Reply:
Good call! Added a comment here describing the difference between this tag parser and the one used for other modes